demo/server/package.json

```diff
@@ -35,16 +35,16 @@
     "start": "node ."
   },
   "devDependencies": {
-    "@ceramic-solutions/ceramic-utils": "workspace:^",
     "@ceramic-solutions/did-utils": "workspace:^",
-    "@ceramic-solutions/points": "workspace:^",
+    "@ceramic-solutions/points": "^0.1.1",
```

Comment:
Better keep using `worskpace:^` here to ensure the package is resolved from the workspace, otherwise pnpm will load it from the npm registry which can cause issues with the local packages.

Reply:
@PaulLeCam good note - this has been updated